dist/dialog.linker.build.js

```diff
@@ -150,26 +170,24 @@ window.jQuery = window.jQuery || window.shoestring;
 			var link = $a.is( "[data-dialog-link]" );
 			var iframe = $a.is( "[data-dialog-iframe]" );
 
-
 			function createDialog(content){
-				var linkId = $a.attr( "id" );
+				var linkHref = $a.attr( "href" );
 				var dialogClasses = $a.attr( "data-dialog-addclass" ) || "";
+				var dialogNoHistory = $a.is( "[data-dialog-nohistory]" );
 				var id;
 
-				if( linkId ) {
-					id = linkId + "-dialog";
-				} else {
-					id = "dialog-" + new Date().getTime();
+				if( linkHref ) {
+					id = linkHref;
```

Comment:
Yes. Which seemed an odd requirement for one, but it gave us a "unique" hash to work with once the dialog was created. Still, this seems more direct I think, using the page source.

Thoughts?